Build error on 'make volume-manager'

**Describe the bug**
I follow the guide [https://github.com/tkestack/volume-decorator#build](url) to build the binary, but something wrong on 'make volume-manager':

```log
~/gopath/src/tkestack.io/volume-decorator]# make volume-manager
./hack/update-codegen.sh
go: finding tkestack.io/tapp v0.0.0-20191105143848-5edda2f9f6c7
go: tkestack.io/tapp@v0.0.0-20191105143848-5edda2f9f6c7: unexpected status (https://mirrors.tencent.com/go/tkestack.io/tapp/@v/v0.0.0-20191105143848-5edda2f9f6c7.info): 404 Not Found
go: error loading module requirements
make: *** [generate] Error 1
```

**Environment**
 - OS: Linux VM_149_11_centos 3.10.107-1-tlinux2_kvm_guest-0049 #1 SMP Tue Jul 30 23:46:29 CST 2019 x86_64 x86_64 x86_64 GNU/Linux
 - golang: go version go1.12.4 linux/amd64

***further debug***
just comment the line of "tkestack.io/tapp" out in `go.mod`, then `make volume-manager ` command runs into an error as:
```log
~/gopath/src/tkestack.io/volume-decorator(build-error-fix)# make volume-manager
./hack/update-codegen.sh
Generating deepcopy funcs
Generating clientset for storage:v1 at tkestack.io/volume-decorator/pkg/generated/clientset
Generating listers for storage:v1 at tkestack.io/volume-decorator/pkg/generated/listers
Generating informers for storage:v1 at tkestack.io/volume-decorator/pkg/generated/informers
go fmt ./pkg/... ./cmd/...
go: finding tkestack.io/tapp latest
go vet ./pkg/... ./cmd/...
files=$(find . -name '*.go' | egrep -v './vendor|generated'); \
revive -config build/linter/revive.toml -formatter friendly $files
/bin/sh: line 1: revive: command not found
make: *** [revive] Error 127
```

--- a/README.md
+++ b/README.md
@@ -10,6 +10,15 @@ Controller that maintains more runtime information for Kubernetes volume(PVC), s
 - Collect current mounted nodes of a volume.
 - Collect real usage bytes of a volume.
 
+## Prerequisites
+These build instructions assume you have a Linux build environment with:
+
+-  Docker
+-  git
+-  make
+-  golang  version > 1.11, with GO111MODULE=on
+-  revive  `go get -u github.com/mgechev/revive`
+
 ## Build
 
 To make the binary, just run:

--- a/build/docker/Dockerfile
+++ b/build/docker/Dockerfile
@@ -2,8 +2,7 @@ FROM centos:7
 LABEL description="TKE Volume Decorator"
 
 ENV CEPH_VERSION "mimic"
-RUN export http_proxy=http://proxy.tencent.com:8080 && https_proxy=http://proxy.tencent.com:8080 && \
-    yum  install -y centos-release-ceph && \
+RUN yum  install -y centos-release-ceph && \
     yum install -y kmod ceph-common ceph-fuse attr && \
     yum clean all && \
     unset http_proxy && unset https_proxy

--- a/go.mod
+++ b/go.mod
@@ -41,5 +41,5 @@ require (
 	sigs.k8s.io/controller-runtime v0.1.10
 	sigs.k8s.io/testing_frameworks v0.1.1 // indirect
 	sigs.k8s.io/yaml v1.1.0 // indirect
-	tkestack.io/tapp v0.0.0-20191105143848-5edda2f9f6c7
+	github.com/tkestack/tapp master
 )

